.github/workflows/release_windows_pytorch_wheels.yml

```diff
@@ -83,13 +83,10 @@ jobs:
       fail-fast: false
       matrix:
         python_version: ["3.11", "3.12", "3.13"]
-        # TODO(#1677): Get 2.9 build working and enable it
-        # pytorch_version: ["release/2.9", "nightly"]
-        pytorch_version: ["nightly"]
+        pytorch_version: ["release/2.9", "nightly"]
         include:
-          # TODO(#1677): Get 2.9 builds working and enable them
-          # - pytorch_version: release/2.9
-          #   pytorch_patchset: rocm_2.9
+          - pytorch_version: release/2.9
+            pytorch_patchset: rocm_2.9
```

Comment:
We could also use [`release/2.9_rocm7.9`](https://github.com/ROCm/pytorch/tree/release/2.9_rocm7.9) (and update the docs) if the cherry-picks aren't merged to [`release/2.9`](https://github.com/ROCm/pytorch/tree/release/2.9)

Reply:
Confirmed that the dev builds passed and https://d25kgig7rdsyks.cloudfront.net/v2/gfx1151/torch/ (dev release index) shows torch-2.9.0 alongside the old torch-2.9.0a0 builds:

```
torch-2.7.0a0+rocm7.0.0.dev0.e96d36b9b628476463ef6cecee752f601052a4d2-cp313-cp313-win_amd64.whl
torch-2.9.0+rocm7.10.0a20251017-cp311-cp311-win_amd64.whl
torch-2.9.0+rocm7.10.0a20251017-cp312-cp312-win_amd64.whl
torch-2.9.0+rocm7.10.0a20251017-cp313-cp313-win_amd64.whl
torch-2.9.0a0+rocm7.0.0rc20250804-cp312-cp312-win_amd64.whl
```